.github/workflows/ci_test-conda.yml

```diff
@@ -11,6 +11,10 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
   cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
 
+defaults:
+  run:
+    shell: bash -l {0}
```

Comment:
q: What does this do?

Reply:
it is replacement for having `shell: bash -l {0}` in each `run` also recommended the action page we are using here